### PR TITLE
remove deprecated env method, bump version to 1.3.0

### PR DESCRIPTION
**Pull request by dorianmariecom**
it's been deprecated for a while

### CODE REVIEW

**Comment by stefannibrasil** [comment]
Thanks @dorianmariecom 🎉 Could you fix the conflict? This should be good to go once the conflict is resolved.

**Comment by dorianmariecom** [comment]
@stefannibrasil done

**Comment by stefannibrasil** [comment]
Thank you! Will cut a release this week, hopefully.

**Review by nickcharlton — COMMENTED** [review]
I missed actually submitting this before it was merged, for some reason. But we should avoid doing this in future.

File: lib/climate_control/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module ClimateControl
-  VERSION = "1.2.0".freeze
+  VERSION = "1.3.0".freeze
```
[the remark this answers is not in the document]
Comment (stefannibrasil) [reply lib/climate_control/version.rb:2]: Ah, I am sorry, Nick! I saw that this method had been deprecated for a while and didn't see any issues. I was planning on cutting a release announcing the deprecation. I will rollback the commit and reopen this PR.

**Pull request merged**